test/core/vm/CMakeLists.txt

```diff
@@ -3,6 +3,10 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}
+        ${PROJECT_SOURCE_DIR}/core
```

Comment:
Not sure I understand it. Why do you include /core into test/core/vm?

Reply:
This is necessary to include VMContext Mock

Reply:
Please, left TODO to remove it after VMContext is implemented.